concepts/Common/Licensing/20 Sharing Keys by Multiple Developers.md

```diff
@@ -0,0 +1,5 @@
+According to the EULA, every developer who works with DevExtreme must have a license, since the DevExtreme license has a per seat basis.
+
+If you have many licenses that you distribute among developers, then each developer will have access to their own personal key in Download Manager under their account.
```

Comment:
```suggestion
If you have many licenses you can assign them to different developers, so that each developer will have their own personal key in Download Manager under their account.
```